lib/services/http-service.js

```diff
@@ -138,18 +139,20 @@ function httpServiceFactory(
             appRoot: path.resolve(__dirname, '../../'),
             configDir: path.resolve(__dirname, '../../swagger_config'),
             swagger: path.resolve(__dirname, '../../static/' + endpoint.yamlName),
+            authEnabled: this.endpoint.authEnabled
         };
 
         var redfishConfig = {
             appRoot: path.resolve(__dirname, '../../'),
             configDir: path.resolve(__dirname, '../../swagger_config'),
             swagger: path.resolve(__dirname, '../../static/redfish.yaml'),
+            authEnabled: this.endpoint.authEnabled
         };
 
         var swaggerCreateAsync = Promise.promisify(swagger.create);
 
         var apiV2 = swaggerCreateAsync(swaggerConfig).then(function(swaggerExpress) {
```

Comment:
Do we do any chaining where we need to return the results of `swaggerExpress.register(app)`? any inadvertent implications to this change, or benefits?